src/components/TrackTile/TrackTile.tsx

```diff
@@ -25,12 +26,17 @@ export type TrackTileProps = {
 export function TrackTile({
   onOpenSettings,
   onOpenTracker,
-  hideSettingsButton = false,
+  hideSettingsButton,
   title = '',
   styles: instanceStyles,
   shouldUseOntology,
 }: TrackTileProps) {
   const { styles } = useStyles(defaultStyles, instanceStyles);
+  const { componentProps } = useDeveloperConfig() || {};
+  if (hideSettingsButton === undefined) {
+    hideSettingsButton = componentProps?.TrackTile?.hideSettingsButton;
+  }
+
```

Comment:
Since developerConfig sets the `componentProp` for all potential `TrackTile` components, the local prop takes precedence.